app/src/main/java/com/xmartlabs/template/controller/SessionController.kt

```diff
@@ -0,0 +1,8 @@
+package com.xmartlabs.template.controller
+
+import com.xmartlabs.bigbang.core.controller.SessionController
+import com.xmartlabs.template.model.Session
+
+val SessionController.session
+    get() = abstractSession as? Session?
```

Comment:
Because this is not an initialization, this queries the actual value each time and returns it. An initialization will just cache the first value